bucketeer2/src/main/kotlin/io/bucketeer/sdk/android/internal/event/SendEventsResult.kt

```diff
@@ -0,0 +1,8 @@
+package io.bucketeer.sdk.android.internal.event
+
+import io.bucketeer.sdk.android.BKTException
+
+sealed class SendEventsResult {
+  data class Success(val sent: Boolean) : SendEventsResult()
```

Comment:
this value will be used to decide whether or not to reschedule in background/foreground polling.